spec/spec_helper.rb

```diff
@@ -1,3 +1,9 @@
 $LOAD_PATH.unshift File.expand_path('../../lib/win32', __FILE__)
 
-require 'win32-certstore'

```

Comment:
We can keep `require 'win32-certstore' if windows?` in this file only. This way we won't have to require all files of `win32-certstore` in each and every spec file for windows. We could just use `require 'spec_helper'`.